hledger/Hledger/Cli/Commands/Includes.hs

```diff
@@ -0,0 +1,57 @@
+{-|
+
+The @includes@ command lists account names:
+
+- by default, it shows names of all the included journals and journal itself.
+
+- with '--omit-self', it shows names of all included files only
```

Comment:
Is the option worthwhile ? I guess the main journal will always be the first item ?

Reply:
not sure if it is worthwhile honestly. Let me sleep on this ;)